### DISCUSSION TITLE
filter the elements visible on the track

### DISCUSSION THREAD

**Discussion by LankyAlec** [GMOD/jbrowse-components#3983] [Ideas] [answered]
Is it possible to filter the elements of a track to see based on one or more specific attributes?
for example I have a track in which both males and females are present and I want to temporarily see only the males.
in the image you see the attributes of the track elements
<img width="562" alt="Screenshot 2023-10-16 alle 10 34 52" src="https://github.com/GMOD/jbrowse-components/assets/66473605/14d372e4-401b-4ffc-9e46-fc6d94f17d89">

**Comment by cmdcolin** [comment, marked as answer]
currently there is not a way to generically apply a filter. I think this would be a great project though, so it would be great to see this get progress. similar github issue here https://github.com/GMOD/jbrowse-components/issues/1988
